### PR TITLE
assignment 34 completed, topics and posts

### PR DESCRIPTION
**Pull request by droberts84**

### CODE REVIEW

File: app/controllers/summaries_controller.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class SummariesController < ApplicationController
+  
+  def show
+    set_topic
+    set_post
+    set_summary
```
Comment (droberts84) [inline app/controllers/summaries_controller.rb]: Once you get the summary, is there anyway you could drive the appropriate `post` and `topic` without having to do a database query or calling the `set_topic`, etc.

**Comment by droberts84** [comment]
Oh I didn't mean for you to merge that!  I just made the Pull Request as it
is an easier form for discussion.  I'll help you fix it a bit later today.

**Comment by brendanmcgivern** [comment]
Ooops.  I saw that the checkpoint hadn't been checked off yet and then I saw the pull request, so I thought you wanted me to merge and make some alterations based on your previous comments.  Sorry about that.  Date: Thu, 28 May 2015 05:55:30 -0700

**Comment by brendanmcgivern** [comment]
I fixed the the accidental merge.  I reverted the pull request on github and then ran git fetch and git pull in my local environment and everything is back to normal.From: bmcg0707@hotmail.com
To: reply@reply.github.com
Subject: RE: [bloccit] assignment 34 completed, topics and posts (#1)
Date: Thu, 28 May 2015 09:06:17 -0400




Ooops.  I saw that the checkpoint hadn't been checked off yet and then I saw the pull request, so I thought you wanted me to merge and make some alterations based on your previous comments.  Sorry about that.  

Date: Thu, 28 May 2015 05:55:30 -0700

**Comment by droberts84** [comment]
Cool!  I forgot that GitHub added the revert feature.  Be aware there are
some surprising consequences form using git revert, but they won't effect
you in this situation.  I'll tell you more about it on Friday.

**Pull request merged**